### ISSUE TITLE
Composer install step silently fails

### ISSUE THREAD

**Issue by jakobbuis** [OskarStark/phpstan-ga#45]
**Steps to reproduce:**
1. Create a composer-application with requirements:
    ```json
    {
        "require": {
            "php": "^7.3",
            "nunomaduro/larastan": "^0.7.5"
        }
    }
    ````
    We don't actually need larastan here, but it introduces a configuration file in vendor that we are going to load.
2. Create a phpstan.neon file in the root directory with:
    ```yaml
    includes:
     - vendor/nunomaduro/larastan/extension.neon
   ```
3. Run the github action

**Expected:** an error message indicating that the composer install failed
**Happened:** an error message indicating that the vendor/nunomaduro/larastan/extension.neon file is missing.

Took me a bit of fiddling with the paths to debug it, but when you run the container locally, it actually throws an error that the composer install command failed, and then happily proceeds to run the analysis. I think it might be better if we add a check on whether composer install actually succeeded, and then bug out immediately? :)

**Comment by dbu** [comment]
similar problem here: https://github.com/rokka-io/rokka-client-bundle/runs/4883258441?check_suite_focus=true

we do install the dependencies as its own step, so the analysis then actually works. but failing the composer step should fail the build, it hides problems otherwise.
